src/Main.purs

```diff
@@ -195,18 +196,18 @@ runBundle cliRoot args = do
     , "      (compile-whole-program \"" <> mainWpoPath <> "\" \"" <> outPath <> "\")"
     , ")))"
     ]
-  case res of
-    Left err -> liftEffect $ Process.exit (fromMaybe 1 err.exitCode)
-    Right _ -> Console.log $ "Created " <> outPath
+  case res.exitCode of
+    Just 0 -> Console.log $ "Created " <> outPath
+    _ -> liftEffect $ Process.exit' (fromMaybe 1 res.exitCode)
```

Comment:
Using `res.exit` will get you an Either-like type containing either the exit code or the kill signal used to kill the child process.

Reply:
Does it matter if I just want to know if it exited with 0? Or will it not emit an exit code if there was a kill signal?

Reply:
No it does not. `exit` just handles the otherwise type unsafe way of doing this:
```
case res.exitCode, res.signal of
  Just c, _ ->
  _, Just s ->
  _, _ -> unsafeCrashWith "Impossible"
```